plugin/externalPluginRoot_test.go

```diff
@@ -36,14 +40,14 @@ func (suite *ExternalPluginRootTestSuite) TestInit() {
 	suite.Regexp(regexp.MustCompile("stdout"), err)
 
 	// Test that Init properly decodes the root from stdout
-	stdout := "{\"name\":\"foo\",\"supported_actions\":[\"list\"]}"
+	stdout := "{\"name\":\"foo\",\"wmethods\":[\"list\"]}"
```

Comment:
How does this test pass? I don't see anything parsing `wmethods`.

Reply:
Huh, that is weird. Not sure. I'll double check this.

Reply:
Oh, forgot to address this. One second.